integreat_cms/cms/templates/analytics/__page_hix_row.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+{% load rules %}
+{% load content_filters %}
+{% load page_filters %}
+{% with page_translation.page.id as page_id %}
+    {% with page_translation.hix_score as hix %}
+        <tr class="hover:bg-gray-100 {% if hix < hix_threshold %}text-red-500{% endif %}">
+            <td>
+                <a href="{% url 'edit_page' region_slug=request.region.slug language_slug="de" page_id=page_id %}">
```

Comment:
```suggestion
                <a class="block p-1"
                   href="{% url 'edit_page' region_slug=request.region.slug language_slug="de" page_id=page_id %}">
```
Maybe a little bit of padding?